src/types/DataTypes.sol

```diff
@@ -89,15 +89,15 @@ library Batch {
 }
 
 library MerkleLockup {
-    /// @notice Struct encapsulating the base constructor parameter of a {SablierV2MerkleLockup} contract.
-    /// @param initialAdmin The initial admin of the Merkle Lockup contract.
-    /// @param asset The address of the streamed ERC-20 asset.
+    /// @notice Struct encapsulating the base constructor parameters of a MerkleLockup campaign.
+    /// @param initialAdmin The initial admin of the MerkleLockup campaign.
+    /// @param asset The contract address of the ERC-20 asset to be distributed.
     /// @param ipfsCID The content identifier for indexing the contract on IPFS.
     /// @param name The name of the campaign.
     /// @param merkleRoot The Merkle root of the claim data.
-    /// @param expiration The expiration of the streaming campaign, as a Unix timestamp.
-    /// @param cancelable Indicates if each stream will be cancelable.
-    /// @param transferable Indicates if each stream NFT will be transferable.
+    /// @param expiration The expiration of the campaign, as a Unix timestamp.
+    /// @param cancelable Indicates if the stream NFT will be cancelable after claiming.
```

Comment:
I think "Indicates if the stream will be cancelable after claiming" is more appropriate.